specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2022-03-08-preview/Backups.json

```diff
@@ -0,0 +1,220 @@
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ServerBackupListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ServerBackupProperties": {
+      "type": "object",
+      "properties": {
+        "backupType": {
+          "type": "string",
+          "description": "Backup type."
+        },
+        "completedTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Backup completed time (ISO8601 format)."
+        },
+        "source": {
+          "type": "string",
+          "description": "Backup source"
+        }
+      },
+      "description": "The properties of a server backup."
+    },
+    "ServerBackup": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/ServerBackupProperties",
+          "x-ms-client-flatten": true,
+          "description": "The properties of a server backup."
+        },
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData",
+          "readOnly": true,
+          "description": "The system metadata relating to this resource."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "description": "Server backup properties"
+    },
+    "ServerBackupListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ServerBackup"
+          },
+          "description": "The list of backups of a server."
+        },
+        "nextLink": {
+          "description": "The link used to get the next page of operations.",
+          "type": "string"
+        }
+      },
+      "description": "A list of server backups."
+    },
+    "CloudError": {
+      "type": "object",
+      "x-ms-external": true,
+      "properties": {
+        "error": {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/ErrorResponse"
+        }
+      },
+      "description": "An error response from the Batch service."
+    }
```

Comment:
This looks like it wraps up the standard ErrorResponse inside another object's "error" proeprty, which would make it non-standard?

Reply:
Used existing v3 type ErrorResponse